packages/template-retail-react-app/.eslintrc.js

```diff
@@ -6,5 +6,20 @@
  */
 
 module.exports = {
-    extends: [require.resolve('pwa-kit-dev/configs/eslint')]
+    extends: [require.resolve('pwa-kit-dev/configs/eslint')],
+    plugins: ['no-relative-import-paths'],
+    rules: {
+        // https://github.com/MelvinVermeer/eslint-plugin-no-relative-import-paths
+        'no-relative-import-paths/no-relative-import-paths': [
+            'error',
+            {
+                allowSameFolder: false,
+                ...(process.cwd().endsWith('template-retail-react-app')
+                    ? {}
+                    : // Otherwise, assumes that the current working directory is the monorepo
+                      {rootDir: 'packages/template-retail-react-app/'}),
+                prefix: 'retail-react-app'
+            }
+        ]
+    }
```

Comment:
This ensures we fail all relative imports in `eslint` checks and ultimately gives us the ability to autocorrect them via `lint:fix`. Relative imports will be disallowed due to how the overrides system works